torchao/quantization/GPTQ.py

```diff
@@ -1022,13 +1024,15 @@ def __init__(
         precision: torch.dtype = torch.float32,
         scales_precision: torch.dtype = torch.float32,
         device: torch.device = torch.device("cpu"),
+        mapping_type: MappingType = MappingType.SYMMETRIC_NO_CLIPPING_ERR
```

Comment:
it looks like this one is still not updated, is this intentional?

Reply:
Oops, updated.